fratikcoiny/src/main/java/pl/fratik/fratikcoiny/libs/chinczyk/Chinczyk.java

```diff
@@ -1482,13 +1494,13 @@ public void shutdown(ChinczykStateDao stateDao) {
             String key;
             if (status == Status.IN_PROGRESS) key = "chinczyk.shutting.down.saving";
             else key = "chinczyk.shutting.down";
-            message.editMessage(t.get(l, key)).override(true).complete();
+            message.editMessage(t.get(l, key)).setReplace(true).complete();
             for (Player p : players.values()) {
                 if (p.isPlaying() && p.getControlHook() != null) {
                     String controlKey;
                     if (status == Status.IN_PROGRESS) controlKey = "chinczyk.control.shutting.down.saving";
                     else controlKey = "chinczyk.control.shutting.down";
-                    p.getControlHook().editOriginal(new MessageBuilder(t.get(p.getLanguage(), controlKey)).build()).queue();
+                    p.getControlHook().editOriginal(MessageEditData.fromContent(t.get(p.getLanguage(), controlKey))).queue();
```

Comment:
editOriginal z wiadomością właśnie działał jak override, więc wszędzie gdzie było editOriginal(Message) musi być setReplace xD
```suggestion
                    p.getControlHook().editOriginal(MessageEditData.fromContent(t.get(p.getLanguage(), controlKey)).setReplace(true)).queue();
```